wsi/drm/drmplane.cpp

```diff
@@ -418,6 +459,21 @@ void DrmPlane::SetInUse(bool in_use) {
   in_use_ = in_use;
 }
 
+bool DrmPlane::IsSupportedModifier(uint64_t modifier, uint32_t format) {
+  uint32_t count = formats_modifiers_.size();
+  for (uint32_t i = 0; i < count; i++) {
+    format_mods obj = formats_modifiers_.at(i);
```

Comment:
this can be const reference, const format_mods& obj =

Reply:
will fix.